lib/twitter_cldr/js/renderers/numbers/rbnf/rbnf_test_helper_renderer.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module TestHelpers
+
+        class RBNFTestHelperRenderer < TwitterCldr::Js::Renderers::Base
+          set_template "mustache/numbers/rbnf/rbnf_test_helper.coffee"
+
+          def global_resource
+            result = {}
```

Comment:
I'd recommend using `inject` instead of each here, just like you did in `number_data_reader_renderer.rb`. Once we completely drop Ruby 1.8 support, you can even use `each_with_object` that doesn't require you to return the container on each iteration and makes things even simplier.